source/common/config/utility.h

```diff
@@ -121,6 +121,13 @@ class Utility {
    */
   static void checkFilesystemSubscriptionBackingPath(const std::string& path);
 
+  /**
+   * Check the grpc_services and cluster_names for API config sanity. Throws on error.
```

Comment:
You can use Doxygeen `@throw` syntax here, grep around for examples.

Reply:
Done in 421a420.